src/sagemaker/local/image.py

```diff
@@ -668,3 +671,13 @@ def _ecr_login_if_needed(boto_session, image):
 
     cmd = "docker login -u AWS -p %s %s" % (token, ecr_url)
     subprocess.check_output(cmd, shell=True)
+
+    return True
```

Comment:
Can you make the other returns explicit False ? We may run into weird stuff if we return None.

Reply:
good point.

Reply:
remove this return. it is only used for test assert and has no other use. subprocess.check_output will throw if there is a real error, so the return value isn't even needed for tests.

Reply:
Don't we need this return to know whether or not to pull as defined here: https://github.com/aws/sagemaker-python-sdk/pull/455/files#diff-04b98903463aa4dcc1df3eafda085121R121